zerver/models.py

```diff
@@ -1369,8 +1369,18 @@ class Stream(models.Model):
     # Whether this stream's content should be published by the web-public archive features
     is_web_public = models.BooleanField(default=False)  # type: bool
 
-    # Whether only organization administrators can send messages to this stream
-    is_announcement_only = models.BooleanField(default=False)  # type: bool
+    STREAM_POST_POLICY_EVERYONE = 1
+    STREAM_POST_POLICY_ADMINS = 2
+    STREAM_POST_POLICY_RESTRICT_NEW_MEMBERS = 3
+    # TODO: Implement policy to restrict posting to a user group or admins.
```

Comment:
This TODO should have been removed in the second commit, right?  I think it's probably correct to just delete this line from the first commit; the codebase is broken with just the first commit and not the second, so realistilcally we're going to squash them (though see my other comments for things we can split out).

Reply:
Oh, I added this TODO as in the issue within brackets it said that for future work the stream writing ability may be restricted to user groups.

Reply:
OK.